docker-compose/docker-compose.yaml

```diff
@@ -172,6 +219,9 @@ services:
       - sourcegraph
     restart: always
     hostname: gitserver-0
+    depends_on:
```

Comment:
What's the reasoning behind adding all these new `depends_on` settings? I'm not 100% clear on the dependency order but is it possible we're getting in a loop here, where frontend-internal can't be healthy until some of these services are healthy, and vice-versa?

Regardless, I don't think the dependency should be added for Prometheus/Grafana since those are definitely unrelated to frontend-internal.

https://docs.sourcegraph.com/dev/background-information/architecture